<?xml version="1.0"?>
<launch>

  <node name="visp_naoqi_ros" pkg="visp_naoqi_ros" type="visp_naoqi_ros" output="screen">
      <rosparam file="$(find visp_naoqi_ros)/config/robot_config.yaml" command="load"/>
      <param name="cmd_vel_topic_name" value="/cmd_vel" />
<!--       <param name="TFdataFile" value= "$(find visp_naoqi_ros)/data/transformations.xml" /> -->
      <param name="TFdataFile" value= "$(find visp_naoqi_ros)data/objects/tabasco_green/grasping_pose_blob.xml" />

  </node>


</launch>
